targets/targets.json

```diff
@@ -9257,14 +9257,15 @@
         "features": ["BLE"],
         "components_add": ["WHD", "43012", "CYW43XXX"],
         "components_remove": ["QSPIF"],
-        "device_has_remove": ["ANALOGOUT", "CRC", "TRNG", "QSPI"],
```

Comment:
Can you add information why are this being removed to the commit message?

Reply:
Updated commit message to add details